playground/nuxt.config.ts

```diff
@@ -13,5 +13,11 @@ export default defineNuxtConfig({
   },
   modules: [
     nuxtMetaModule
-  ]
+  ],
+  componentMeta: {
+    checkerOptions: {
+      forceUseTs: true,
+      schema: {}
```

Comment:
`schema` is required in order to extract the schema, maybe we should use `defu` to have default options?

Reply:
We can provide default value using `defaults` key


```ts
export default defineNuxtModule<ModuleOptions>({
  meta: {
    name: 'nuxt-component-meta',
    configKey: 'componentMeta'
  },
  defaults: {
    checkerOptions: {}
  },
  setup() { ...}
})
```